releasenotes/notes/add-bksf-mapper-7826652bfa4403a6.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Add :class:`qiskit_nature.mappers.second_quantization.BravyiKitaevSFMapper` implementing the
```

Comment:
In case you do go with my suggested name change, be sure to update it here, too.

Reply:
Done